### PR TITLE
Adjust limits

### PR DESCRIPTION
**Pull request by tokle**

### CODE REVIEW

File: tests/performance/container/http/container-http.rb [hunk 1]
```diff
@@ -95,26 +95,26 @@ def test_container_http_performance_with_logging
             :filter => {'legend' => STANDARD},
             :x => 'legend',
             :y => 'qps',
-            :y_min => 110000,
-            :y_max => 140000,
+            :y_min => 63000,
```
Comment (baldersheim) [inline tests/performance/container/http/container-http.rb:98]: Why this drop when using persistent connections ? I thought connection logging only took place when connection was established/disconnected.

Comment (tokle) [reply tests/performance/container/http/container-http.rb:98]: This is the baseline with access logging enabled. Connection logging is not enabled yet.

Comment (baldersheim) [reply tests/performance/container/http/container-http.rb:98]: K

**Review by baldersheim — APPROVED** [review]

**Review by bjorncs — COMMENTED** [review]
👍

**Pull request merged**